frontend/src/Components/Calendly.js

```diff
@@ -0,0 +1,145 @@
+    width: "245px",
+    borderRadius: "10px",
+    textTransform: "none",
+    fontSize: "20px",
+  },
+});
+
+const Calendly = () => {
+  const classes = useStyles();
+  //   const { token } = useContext(AuthContext);
+  //   const decoded = jwt_decode(token);
+  //   const [userId, setUserId] = useState(decoded.userId);
+  const [calendlyURL, setCalendlyURL] = useState("");
+  const [isValid, setIsValid] = useState(false);
+
+  //   const getUserProfile = async () => {
+  //     try {
+  //       let config = {
+  //         method: "get",
+  //         maxBodyLength: Infinity,
+  //         url: `${api.users.userProfile}/${userId}`,
+  //         headers: {
+  //           Authorization: `Bearer ${token}`,
+  //           "Content-Type": "application/json",
+  //         },
+  //       };
+  //       const response = await axios.request(config);
+
+  //       const { calendlyURL } = response.data;
+  //       setCalendlyURL(calendlyURL);
+  //       setIsValid(validateInput(calendlyURL));
+  //     } catch (error) {
+  //       console.error(error);
+  //     }
+  //   };
+
+  //   useEffect(() => {
+  //     getUserProfile();
+  //   }, []);
+
+  const handleURLChange = (event) => {
+    setCalendlyURL(event.target.value);
+    setIsValid(validateInput(event.target.value, calendlyURL));
+  };
+  const validateInput = (calendlyURL) => {
+    const regex =
+      /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/i;
+    return regex.test(calendlyURL);
+  };
+
+  //   const handleContinueClick = async () => {
+  //     try {
+  //       let data = JSON.stringify({
+  //         calendly,
+  //       });
+  //       let config = {
+  //         method: "patch",
+  //         maxBodyLength: Infinity,
+  //         url: `${api.users.userProfile}/${userId}`,
+  //         headers: {
+  //           Authorization: `Bearer ${token}`,
+  //           "Content-Type": "application/json",
+  //         },
+  //         data: data,
+  //       };
+  //       const response = await axios.request(config);
+  //       console.log(response.data);
+  //       // move to next step of questionnaire
+  //     } catch (error) {
+  //       console.error(error);
+  //     }
+  //   };
+
+  return (
+    <>
+      <Nav showLogoutButton={true} />
+      <div className="progress-bar-form-container">
+        <LinearDeterminate page={3} />
+        <h1 className="welcome">Let's get you matched!</h1>
+        <h2 className="tellus">
+          Enter your Calendly URL so that mentees can schedule time with you.
+        </h2>
+        <h3>Calendly URL</h3>
+        <div className="input-container">
+          <TextField
+            id="outlined-basic"
+            label="Calendly URL"
+            variant="outlined"
+            value={calendlyURL}
+            onChange={handleURLChange}
+            sx={{ width: "671.85px", marginBottom: "2rem" }}
+          />
+          <p>What is Calendly?</p>
```

Comment:
add anchor to the calendly's website, pls look at the ticket: https://github.com/orgs/cherryontech/projects/3/views/1?pane=issue&itemId=24721353